src/client/Microsoft.Identity.Client/AuthScheme/TokenType.cs

```diff
@@ -1,37 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-namespace Microsoft.Identity.Client.AuthScheme
-{
-    /// <summary>
-    /// Specifies the token type to log to telemetry.
-    /// </summary>
-    internal enum TokenType
```

Comment:
Any reason for removing the enum? I liked the enum better, they are more readable and separates out the token types from all the other constants.

Reply:
discussed offline. no longer needed as everything will be an int going forward